cypress/e2e/article.cy.js

```diff
@@ -1,13 +1,17 @@
 describe('', () => {
+  let user;
   before(() => {
-
+    cy.task('generateUser').then((generatedUser) => {
+      user = generatedUser;
+    });
   });
 
-  it('', () => {
-
-  });
-
-  it('', () => {
-
+  it('should provide an ability to create the article', () => {
+    cy.visit('/user/register');
+    cy.login(user);
+    cy.getCookie('auth').should('exist');
+    cy.createArticle().then((articleInfo) => {
```

Comment:
There should be 2 tests in total. 
1. login() -> create an article manually, by clicking on the page -> assert the article has been created
2. login() -> create an article with a function -> delete the article (however you like) -> assert the article has been deleted.

Reply:
fixed